bears/general/RangeHelpers.py

```diff
@@ -22,21 +22,21 @@ def get_specified_block_range(file,
                             block has begun.
     :param close_specifier: A character or string indicating that the block
                             has ended.
-    :param annotation_dict: A dictionary containing sourceranges of all the
-                            strings and comments within a file.
+    :param ranges:          A dict contating ranges of various code
+                            segments.
```

Comment:
less clear than before. I'd not now what to pass to this function to use it, maybe explain the keys?